contact_map/contact_count.py

```diff
@@ -98,19 +100,51 @@ def df(self):
         columns = list(range(self.n_y))
         return pd.SparseDataFrame(mtx, index=index, columns=columns)
 
-    def plot(self, cmap='seismic', vmin=-1.0, vmax=1.0, with_colorbar=True):
+    def _check_number_of_pixels(self, figure):
+        """
+        This checks to see if the number of pixels in the figure is high enough
+        to accuratly represent the the contact map. It raises a RuntimeWarning
+        if this is not the case.
+
+        Parameters
+        ----------
+        figure: :class:`matplotlib.Figure`
+            matplotlib figure to compare the amount of pixels from
+
+        """
+        # Get dpi, and total pixelswidht and pixelheight
+        dpi = figure.get_dpi()
+        figwidth = figure.get_figwidth()
+        figheight = figure.get_figheight()
+        xpixels = dpi*figwidth
+        ypixels = dpi*figheight
+
+        # Check if every value has a pixel
+        if xpixels/self.n_x < 1 or ypixels/self.n_y < 1:
+            msg = ("The number of pixels in the figure is insufficient to show"
+                   " all the contacts.\n Please save this as a vector image "
+                   "(such as a PDF) to view the correct result.\n Another "
+                   "option is to increase the 'dpi' (currently: "+str(dpi)+"),"
+                   " or the 'figsize' (curently: "+str((figwidth, figheight)) +
+                   ").\n Adviced minimum amount of pixels = "
```

Comment:
"Recommended minimum" (better word choice, and typo in current spelling anyway)

Reply:
like this, or without the `amount of pixels`?

Reply:
with `amount of pixels`; I just meant that "Adviced" is wrong